content/en/docs/v3.5/learning/data_model.md

```diff
@@ -24,6 +24,11 @@ The key of key-value pair is a 3-tuple (major, sub, type). Major is the store re
 
 etcd also keeps a secondary in-memory [btree][btree] index to speed up range queries over keys. The keys in the btree index are the keys of the store exposed to user. The value is a pointer to the modification of the persistent b+tree. Compaction removes dead pointers.
 
+Overall, etcd get the revision information from btree and then using the revision as key to fetch value from b+tree(As shown below).
+<p align="center">
+  <kbd><img src="https://user-images.githubusercontent.com/108830687/221443117-6f8c6d51-001a-4ef2-8b66-d8c494a47293.png" alt="mvcc data model"width="80%" height="80%" \></kbd>
```

Comment:
Thanks @miancheng7 for the PR. But where is the link from? Shouldn't the image be put/uploaded to https://github.com/etcd-io/website/tree/main/content/en/docs/v3.6/learning/img?

Reply:
Hi @ahrtr, thanks for pointing it out. This link is automatically generated by Github. I will upload the image to the link you provided and replace the link here in next PR.

Reply:
Could you put the image under [v3.5/learning/img](https://github.com/etcd-io/website/tree/main/content/en/docs/v3.5/learning/img) and follow the existing name convention (e.g. "data-model-figure-01.png") in this PR? It's hard to track everything cross PRs.

It's OK to update 3.6 and 3.4 in separate PRs.

Reply:
Thanks, will do.

Reply:
@ahrtr +1 good suggestion!